File: FDNS.Domain/Models/DomainContacts.cs
using FDNS.Common.Constants;
using System;

namespace FDNS.Domain.Models
{
    public class DomainContacts : BaseEntity<Guid>
    {
        public string OrganizationName { get; set; }
        public string JobTitle { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string City { get; set; }
        public string StateProvince { get; set; }
        public string StateProvinceChoice { get; set; }
        public string PostalCode { get; set; }
        public string Phone { get; set; }
        public string PhoneExt { get; set; }
        public string Fax { get; set; }
        public string EmailAddress { get; set; }
        public ContactsType ContactsType { get; set; }
        public Country Country { get; set; }
        public ushort CountryId { get; set; }
        public Domain Domain { get; set; }
        public Guid DomainId { get; set; }
    }
}

File: client-app/src/app/models/domain/domainContacts.ts
export interface DomainContacts {
    organizationName?: string;
    jobTitle?: string;
    firstName: string;
    lastName: string;
    address1: string;
    address2?: string;
    city: string;
    countryName: string;
    stateProvince: string;
    postalCode: string;
    phone: string;
    phoneExt?: string;
    fax?: string;
    emailAddress: string;
    contactsType?: number;
}